packages/env/lib/env.js

```diff
@@ -170,6 +170,51 @@ module.exports = {
 		spinner.text = 'Stopped WordPress.';
 	},
 
+	/**
+	 * Remove the containers and local volumes.
+	 *
+	 * @param {Object}  options
+	 * @param {Object}  options.spinner A CLI spinner which indicates progress.
+	 * @param {boolean} options.debug   True if debug mode is enabled.
+	 */
+	async remove( { spinner, debug } ) {
+		spinner.info(
+			'WARNING! This will remove local volumes and containers.'
+		);
+		const { yesDelete } = await inquirer.prompt( [
+			{
+				type: 'confirm',
+				name: 'yesDelete',
+				message: 'Are you sure you want to continue?',
+				default: false,
+			},
+		] );
+		if ( yesDelete ) {
```

Comment:
Let's flip this and return early so the code that follows isn't so indented. 

```js
if ( ! yesDelete ) {
	spinner.text = 'Cancelled';
	return;
}
```

Reply:
Good idea!